lib/src/Base/Func/LinearEnumerateFunction.cxx

```diff
@@ -174,6 +174,16 @@ void LinearEnumerateFunction::setUpperBound(const Indices & /*upperBound*/)
   throw NotYetImplementedException(HERE) << " in LinearEnumerateFunction::setUpperBound";
 }
 
+/* The marginal enumerate function */
+EnumerateFunctionImplementation LinearEnumerateFunction::getMarginal(const Indices & indices) const
```

Comment:
```suggestion
EnumerateFunction LinearEnumeateFunction::getMarginal(const Indices & indices) const
```

Reply:
Thank you very much for the detailed suggestions. Fixed at 8ba0bdcdc775a35dbdf8f2987f7701ee352fa488.